libcxx/test/std/strings/basic.string/string.capacity/reserve.deprecated_in_cxx20.verify.cpp

```diff
@@ -10,7 +10,7 @@
 
 // void reserve(); // Deprecated in C++20
 
-// UNSUPPORTED: c++03, c++11, c++14, c++17
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++26
```

Comment:
Maybe rewrite this as a `REQUIRES`?

Reply:
That won't work properly; then the test will no longer be executed in C++29. It would be great when we can have `// UNSUPPORTED: <c++26` instead. IIRC there was some interest in that feature on Discord recently.

Reply:
We don't want this test to run in C++29, or am I missing something?